fractional-NFT/Forc.toml

```diff
@@ -0,0 +1,7 @@
+members = [
+  "./project/fractional-NFT-contract",
+  "./project/fractional-NFT-contract/tests/artifacts/NFT",
```

Comment:
This doesn't build when I `forc build` in the root of the project. Does it build for you?

Reply:
To be more specific, by "this" I mean the NFT project in fractional NFT's artifacts.